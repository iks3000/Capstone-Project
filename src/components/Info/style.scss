@import '../../assets/styles/_extends.scss';

.info {
    @extend %content-text;
}