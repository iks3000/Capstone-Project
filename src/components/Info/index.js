import "./style.scss";

const Info = ({ text }) => {
  return <p className="info">{text}</p>;
};

export default Info;
