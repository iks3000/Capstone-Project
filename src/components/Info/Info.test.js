import { shallow } from "enzyme";
import Info from "./index";
it("should render Inner component", () => {
  const mockItem = {
    text: "test",
  };
  expect(shallow(<Info {...mockItem} />)).toMatchSnapshot();
});
