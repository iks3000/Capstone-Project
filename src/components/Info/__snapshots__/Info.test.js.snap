// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Inner component 1`] = `
<p
  className="info"
>
  test
</p>
`;
